src/neuroconv/tools/data_transfers/__init__.py

```diff
@@ -1,14 +1,6 @@
 """Collection of helper functions for assessing and performing automated data transfers."""
 
-from ._aws import estimate_s3_conversion_cost
 from ._dandi import automatic_dandi_upload
 from ._globus import get_globus_dataset_content_sizes, transfer_globus_content
-from ._helpers import estimate_total_conversion_runtime
 
-__all__ = [
-    "estimate_s3_conversion_cost",
-    "automatic_dandi_upload",
-    "get_globus_dataset_content_sizes",
-    "transfer_globus_content",
-    "estimate_total_conversion_runtime",
-]
+__all__ = ["automatic_dandi_upload", "get_globus_dataset_content_sizes", "transfer_globus_content"]
```

Comment:
```suggestion
__all__ = ["automatic_dandi_upload", "get_globus_dataset_content_sizes", "transfer_globus_content",]
```

Let's keep this thing to keep the diffs simpler if you don't mind (trailing coma)